components/sections/vlmd/quiz.js

```diff
@@ -0,0 +1,273 @@
+import React from "react"
+import Markdown from "@/components/elements/markdown"
+import questions from "./questions.json"
+import FileDownloadIcon from "@mui/icons-material/FileDownload"
+import { Autocomplete, Box, Button, TextField } from "@mui/material"
+
+export function Quiz({
+  state,
+  dispatch,
+  handleSetSelected,
+  handleDownloadResouces,
+}) {
+  const [currentQuestionNumber, setCurrentQuestionNumber] = React.useState(0)
+  const currentRequiredQuestions =
+    state["has-specific-study"] === "1"
+      ? questions
+      : questions.filter((q) => !q.enabledBySpecificStudy)
+  const currentQuestion = currentRequiredQuestions[currentQuestionNumber]
+
+  const canGoBack = currentQuestionNumber > 0
+  const canGoForward =
+    currentQuestionNumber < currentRequiredQuestions.length - 1 &&
+    !isQuestionUnanswered(state, currentQuestion)
+  const isFormClear = Object.values(state).every(
+    (v) => v === null || (Array.isArray(v) && v.length === 0)
+  )
+  const isFormDone =
+    currentQuestionNumber === currentRequiredQuestions.length - 1 &&
+    !isQuestionUnanswered(state, currentQuestion)
+
+  return (
+    <div style={{ flex: "0 0 400px" }}>
+      <div style={{ position: "sticky", top: 0 }}>
+        <Question
+          number={currentQuestionNumber + 1}
+          question={currentQuestion}
+          selected={state[currentQuestion.id]}
+          setSelected={(value) => {
+            handleSetSelected(currentQuestion.id, value)
+          }}
+        />
+
+        <div
+          style={{
+            display: "flex",
+            gap: "1rem",
+            justifyContent: "space-between",
+            color: "#782c5c",
+            fontSize: "1.1em",
+          }}
+        >
+          {canGoBack && (
```

Comment:
i love all these helpers--canGoBack, isQuestionUnanswered, etc.

not sure it matters too much, but using disabled={!canGoBack} would keep UI elements present while indicating functionality, or lack thereof, which might be a preferred experience. we can also consider this if it arises.